lib-ui-compose/src/commonMain/kotlin/com/ustadmobile/libuicompose/view/report/reportedit/ReportEditScreen.kt

```diff
@@ -0,0 +1,331 @@
+                        },
+                        value = seriesItem.reportTimeRange?.value ?: 0,
+                        onOptionSelected = { selectedOption ->
+                            val selectedTimeRange =
+                                ReportTimeRange.entries.firstOrNull { it.value == selectedOption.value }
+                                    ?: ReportTimeRange.LAST_WEEK
+                            val updatedSeries = seriesItem.copy(reportTimeRange = selectedTimeRange)
+                            onSeriesChanged(updatedSeries)
+                        }
+                    )
+                }
+            }
+
+            // Filters Section
+            item {
+                if (!seriesItem.reportSeriesFilters.isNullOrEmpty()) {
+                    Text(
+                        text = stringResource(MR.strings.filters),
+                        modifier = Modifier.defaultScreenPadding()
+                    )
+                }
+                seriesItem.reportSeriesFilters?.forEachIndexed { index, reportFilter2 ->
+                    Row(
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .defaultScreenPadding()
+
+                    ) {
+                        val fieldName = reportFilter2.reportFilterField?.name?.lowercase()
+                            ?.replaceFirstChar { it.uppercase() } ?: ""
+                        val comparisonSymbol =
+                            reportFilter2.reportFilterCondition?.let { getComparisonSymbol(it) }
+                        val filterText =
+                            "$fieldName $comparisonSymbol ${reportFilter2.reportFilterValue}"
+
+                        Text(
+                            text = filterText,
+                            modifier = Modifier.weight(1f)
+                        )
+                        Icon(
+                            imageVector = Icons.Filled.Close,
+                            contentDescription = "Remove filter",
+                            modifier = Modifier
+                                .clickable {
+                                    onRemoveFilter(index, seriesItem.reportSeriesUid)
+                                }
+                        )
+                    }
+                }
+            }
+
+            item {
+                Button(
+                    onClick = { onAddFilter(seriesItem.reportSeriesUid) },
+                    modifier = Modifier.fillMaxWidth()
+                ) {
+                    Text(
+                        text = stringResource(MR.strings.add_filter),
+                    )
+                }
+            }
+        }
+
+        item {
+            Button(onClick = { onAddSeries() }, modifier = Modifier.fillMaxWidth()) {
+                Text(
+                    text = stringResource(MR.strings.add_series),
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun EditReportDropdown(
```

Comment:
Please do **not** create a new dropdown component (EditReportDropDown must be removed). Even UstadExposedDropDownField itself shouldn't really exist.

Please implement dropdowns as per the [Jetpack Compose docs for ExposedDropdownMenuBox](https://developer.android.com/reference/kotlin/androidx/compose/material3/package-summary#ExposedDropdownMenuBox(kotlin.Boolean,kotlin.Function1,androidx.compose.ui.Modifier,kotlin.Function1))